### ISSUE TITLE
Löve 11.3 crashes after drawing 15000 lines

### ISSUE THREAD

**Issue by argot42** [love2d/love#1698]
Hi everybody, 
I'm trying to draw an hexagonal grid with `love.graphics.line` but after 15000 lines the program crashes.
Although this doesn't happen if I set the env variable `LOVE_GRAPHICS_USE_GL2=1`.

This is the code
```
function love.load()
    hexradius = 10
    size = 50

    hexgrid = newGrid(hexradius, size, size) 
end

function love.draw(dt)
    drawgrid()
end

function drawgrid()
    local jxOffset = hexgrid.rad * -math.tan(math.pi/1.5) 
    local ixOffset = jxOffset/4
    local iyOffet = jxOffset * math.sin(math.pi/3)


    for i=1,hexgrid.size.x do
        for j=1,hexgrid.size.y do
            love.graphics.push()
        
            love.graphics.translate(ixOffset + j * jxOffset, i * iyOffet)
            love.graphics.line(
                hexgrid.hex[1].x, hexgrid.hex[1].y,
                hexgrid.hex[2].x, hexgrid.hex[2].y,
                hexgrid.hex[3].x, hexgrid.hex[3].y,
                hexgrid.hex[4].x, hexgrid.hex[4].y,
                hexgrid.hex[5].x, hexgrid.hex[5].y,
                hexgrid.hex[6].x, hexgrid.hex[6].y,
                hexgrid.hex[1].x, hexgrid.hex[1].y)

            love.graphics.pop()
        end

        ixOffset = -ixOffset
    end
end

function newGrid(rad, xsize, ysize)
    local g = {
        rad = rad,
        hex = {},
        size = {
            x = xsize,
            y = ysize,
        },
    }

    for i=1,6 do
        local dir = math.pi/3 * (i+0.5)

        g.hex[i] = {}
        g.hex[i].x = g.rad * math.cos(dir)
        g.hex[i].y = g.rad * math.sin(dir)
    end

    return g
end
```
I'm using Löve 11.3 and the output of `love.graphics.getRendererInfo()` is
```
OpenGL  4.6 (Compatibility Profile) Mesa 21.1.0 Intel   Mesa Intel(R) UHD Graphics 620 (WHL GT2)
```

Thanks!

**Comment by MikuAuahDark** [comment]
I believe this issue has been fixed in 96b8d29a5c94e7b2f5670276f04834860b0f44bd. If this still happen on `master` branch, then it must be another bug.

**Comment by argot42** [comment]
oh, I might be running an old version, sorry! 
I'm using the .AppImage I downloaded from the site.

**Comment by MikuAuahDark** [comment]
In that case, yeah, this bug is supposed to be fixed in next 11.4 release.
